docs/sources/auth/planning/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Planning IAM integration strategy
+menuTitle: IAM integration
+description: Considerations and .
+aliases:
+  - /docs/mimir/latest/old-architecture/
+  - docs/sources/auth/planning/
+weight: 200
+keywords:
+  - IdP
+  - IAM
+  - Auth
+---
+
+# Planning your IAM integration strategy
+
+The following documentation is meant to shed light on the different authorization and authentications strategies available in Grafana. By doing preparation before implementation, the user will be able to decide which integration strategy suits best for their needs.
+
+Identity and Access Management (IAM) is needed to handle secure access to [Grafana resources]. Integrating Grafana with the proper solution is a key step to allow user access to sensitive data and resources, and to simplify user management and authentication.
+
+## Benefits of having an IAM integration
+
+Integrating with an IAM provider provides the following benefits:
+
+- **User management**: By providing Grafana access to your current user database, you eliminate the overhead of replicating user information and instead have centralized user management for users' roles and permissions to Grafana resources.
+
+- **Security**: IAM solutions provide advanced security features such as MFA, RBCA, and audit trails, which can help to improve the security of your Grafana installation.
+
+- **SSO**: Properly setting up Grafana with your current IAM solution enables users to access Grafana with the same credentials they use for other applications.
+
+- **Scalability**: Adding, updating, or removing users from your user database is immediately reflected in Grafana.
+
+In order to plan an integration with Grafana, assess your organization's current needs, requirements and any existing IAM solutions being used. This should include which set of roles and permissions will be match to each type of user and which groups of users share the same set of permissions and access to shared resources.
```

Comment:
It feels like we've moved out of benefits and into something else. Does the rest of the documentation expand on the points you've made in these two paragraphs?

Reply:
I suppose we could put this content into the intro for now. Definitely feels like that's a better place for it at the moment. After it's there, I'll have additional thoughts.